contracts/protocol/facets/ExchangeHandlerFacet.sol

```diff
@@ -888,11 +875,7 @@ contract ExchangeHandlerFacet is IBosonExchangeHandler, BuyerBase, DisputeBase {
      *
      * @return bool - true if buyer is authorized to commit
      */
-    function authorizeCommit(
-        address _buyer,
-        Offer storage _offer,
-        uint256 exchangeId
```

Comment:
I guess this should start with an underscore, _exchangeId ... but i don't want to block on this ... Can we changed in the next version

Reply:
@anajuliabit